types/chrome/chrome-cast/index.d.ts

```diff
@@ -857,18 +857,18 @@ declare namespace chrome.cast.media {
          */
         constructor(sessionId: string, mediaSessionId: number);
 
-        activeTrackIds: number[];
-        currentItemId: number;
-        customData: Object;
+        activeTrackIds?: number[];
+        currentItemId?: number;
+        customData?: Object;
         idleReason: chrome.cast.media.IdleReason | null;
-        items: chrome.cast.media.QueueItem[];
+        items?: chrome.cast.media.QueueItem[];
```

Comment:
Also should be nullable array of non-nullable...

```suggestion
        items: chrome.cast.media.QueueItem[] | null;
```